os-server/src/main/java/app/openschool/auth/AuthController.java

```diff
@@ -85,4 +88,22 @@ private HttpHeaders getJwtHeader(UserPrincipal userPrincipal) {
     headers.add(JWT_TOKEN_HEADER, TOKEN_PREFIX + jwtTokenProvider.generateJwtToken(userPrincipal));
     return headers;
   }
+
+  @PostMapping("/password/forgot")
+  @Operation(summary = "send token to given email")
+  public ResponseEntity<String> forgotPassword(@RequestBody String email, Locale locale)
+      throws MessagingException, UnsupportedEncodingException {
+    authService.updateResetPasswordToken(email);
+    return ResponseEntity.status(OK)
+        .body(messageSource.getMessage("password.sending.success.message", null, locale));
```

Comment:
Let's try to define a standard error response type for whole project, I think it is not good idea just return a plain string.